.github/workflows/ci-renovate-rbe.yml

```diff
@@ -0,0 +1,53 @@
+name: CI - Renovate - RBE
+
+on:
+  push:
+    branches:
+      - renovate/*
+  workflow_dispatch:
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'seleniumhq'
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Repin Dependencies
+        run: |
+          REPIN=1 bazel run @maven//:pin
+      - name: Run format script
+        run: ./scripts/format.sh
+      - name: Commit files
+        run: |
+          export CHANGES=$(git status -s)
+          if [ -n "$CHANGES" ]; then
+            git config --local user.email "selenium-ci@users.noreply.github.com"
+            git config --local user.name "Selenium CI Bot"
+            git add ./java/maven_install.json
+            git commit -m 'Repin maven dependencies'
+            echo "code_changes=true" >> $GITHUB_ENV
+          fi
+      - name: Push changes
+        if: ${{ env.code_changes }} == 'true'
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.SELENIUM_CI_TOKEN }}
+          branch: ${{ github.head_ref }}
+      - name: Format
+        uses: ./.github/workflows/bazel.yml
```

Comment:
I don't think you can use workflow from step.

Reply:
added it as a separate action which depends on fixing the formatting issue